From b490a2d82583e8b36f082fba24a0637ce0e66493 Mon Sep 17 00:00:00 2001
From: Alessio Morale <alessiomorale@gmail.com>
Date: Sun, 21 Apr 2013 23:14:12 +0200
Subject: [PATCH] Enable rtos for stm32_stlink targets

---
 tcl/target/stm32_stlink.cfg |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tcl/target/stm32_stlink.cfg b/tcl/target/stm32_stlink.cfg
index 7dccd47..d85238a 100644
--- a/tcl/target/stm32_stlink.cfg
+++ b/tcl/target/stm32_stlink.cfg
@@ -55,6 +55,6 @@ transport select $_TRANSPORT
 hla newtap $_CHIPNAME cpu -expected-id $_CPUTAPID
 
 set _TARGETNAME $_CHIPNAME.cpu
-target create $_TARGETNAME hla_target -chain-position $_TARGETNAME
+target create $_TARGETNAME hla_target -chain-position $_TARGETNAME -rtos auto
 
 $_TARGETNAME configure -work-area-phys 0x20000000 -work-area-size $_WORKAREASIZE -work-area-backup 0
-- 
1.7.10.4

